docs/deployment_ops_guide.md

```diff
@@ -0,0 +1,850 @@
+
+Examples:
+```
+# set level for all modules via envvar
+$ explore GOLOG_LOG_LEVEL="debug"
+
+# set levels for multiple modules via envvar (can be different levels per module)
+$ export GOLOG_LOG_LEVEL_NAMED="chain:debug,chainxchg:info"
+
+# set levels for multiple modules via arg on startup
+$ lily daemon --log-level-named="chain:debug,chainxchg:info"
+
+# set levels for multiple modules via CLI (requires daemon to be running, one level per command)
+$ lily log set-level --system chain --system chainxchg debug
+```
+
+#### Jaeger Tracing
+
+Lily is capable of exposing traces during normal runtime. This behavior is disabled by default because there is a performance impact for these traces to be captured. These traces are produced using Jaeger and are compatible with [OpenCensus](https://opencensus.io/tracing/). 
+
+Jaeger tracing can be enabled by passing the `--tracing` flag on daemon startup. There are other configuration values which have "reasonable" default values, but should be reviewed for your use case before enabling tracing. (See [option: Custom tracing configuration (TODO: link needed))[#linkneeded] or execute `lily help monitoring` in the console for more information.)
+
+#### Capturing Profiles
+
+Lily exposes runtime profiling endpoints during normal runtime. This behavior is always available, but waits for interaction through the exposed HTTP endpoint before capturing this data.
+By default, the profiling endpoint is exposed at `http://0.0.0.0:1234/debug/pprof`. This will serve up valid HTML to be viewed through a browser client or this endpoint can be connected to using the `go pprof tool` using the appropriate endpoint for the type of profile to be captured. (See [interacting with the pprof HTTP endpoint](https://pkg.go.dev/net/http/pprof) for more information.)
+
+Example:
+
+Capture local heap profile and load into pprof for analysis:
+
+```
+$ curl 0.0.0.0:1234/debug/pprof/heap -o heap.pprof.out
+$ go tool pprof ./path/to/binary ./heap.pprof.out
+```
+
+Inspect profile interactively via `http://localhost:1234/debug/pprof` and host a web interface at `http://localhost:8000` (which opens automatically once profile is captured):
+
+```
+$ go tool pprof -http :8000 :1234/debug/pprof/heap
+```
+
+## Common Configuration Options
+
+(TODO: I'm expecting there to be overlap w regular Lily docs here. Leaving this for last.)
+
+The options described here are not exhaustive and focus on those most helpful to operator usage. Please refer to the Lily Documentation for full details on each option.
+
+### Configuration Precedence
+
+There are many ways to define certain options in Lily and precedence is defined in the following order:
+
+0. Command line flag value from user
+1. Environment variable (if specified)
+2. Configuration file (if specified)
+3. Default defined on the flag
+
+### Options: `lily init`
```

Comment:
I am not sure, but perhaps, rather than documenting the "options", it may be better to document the concepts (and then say that "this is achieved by setting X option when running lily/task/...".

For example, a section on "Confidence", "Window size", "Persistence formats", "Config files and repository" ...